tests/data_manager.rs

```diff
@@ -689,6 +690,280 @@ fn mutable_data_parallel_mutations() {
     verify_data_is_stored(&mut nodes, &mut clients[0], &all_data);
 }
 
+// Client trying to mutate a mutable_data concurrently with two different sets of actions.
+// The responses to the two mutate attempts shall be:
+//  1, both succeeded, when there is no conflicting mutation
+//  2, only one succeeded, when there is conflicting mutation (updating the same key)
+#[test]
+fn mutable_data_concurrent_mutations() {
+    let seed = None;
+    let node_count = TEST_NET_SIZE;
+    let client_count = 1;
+    let data_count = 5;
+    let iterations = test_utils::iterations();
+
+    let network = Network::new(GROUP_SIZE, seed);
+    let mut rng = network.new_rng();
+    let mut event_count = 0;
+    let mut nodes = test_node::create_nodes(&network, node_count, None, false);
+    let mut clients: Vec<_> = (0..client_count)
```

Comment:
Again, only one client used. This can be simplified to just `let client = TestClient::new(...)`.